apps/docs/content/components/Card.mdx

```diff
@@ -41,6 +41,61 @@ The link label defaults to `Read more` and can be customized using the `label` p
 </Card>
 ```
 
+### Label
+
+Use the `Label` to add metadata to the `Card`. The `Label` is optional and can be used to indicate the status of the content.
```

Comment:
It feels quite odd to me that we've got a `label` prop and a `Label` child which do different things. I don't remember the API exploration for this, but I feel like we should have a child called `Card.Link` or `Card.CTA`, where you can pass your text in directly to be more composeable. It doesn't need to accept a `href`, but would eliminate the need for a parent-level prop, which only applies to a child.

```jsx
<Card href="https://github.com">
  <Card.Label>Limited</Card.Label>
  <Card.Heading>Collaboration is the key to DevOps success</Card.Heading>
  <Card.Description>
    Everything you need to know about getting started with GitHub Actions.
  </Card.Description>
  <Card.Link>Click here</Card.Link>
</Card>
```

Reply:
I can see further down that you've actually renamed this to `linkText`. In that case, the Link text description and example would need to change in the docs.

![Screenshot 2023-03-09 at 14 37 01](https://user-images.githubusercontent.com/13340707/224057641-474f3fce-7926-4e9e-a825-419edaaf5598.png)

I think `linkText` is fine, but also feel that moving to a composeable API for this would be even better [for our users](https://github.com/primer/brand/issues/123).

Reply:
Agree 👍 I'll make it composable